torch/csrc/jit/codegen/cuda/expr_evaluator.h

```diff
@@ -43,15 +43,15 @@ class TORCH_CUDA_API EvaluationContext {
 // Evaluates expressions in a Fusion IR, using the passed in
 // context (EvaluationContext) to query for concrete_values. The
 // evaluation context may override concrete values in the IR as well.
-class TORCH_CUDA_API ExpressionEvaluator : private IterVisitor {
+class TORCH_CUDA_API ExpressionEvaluator : protected IterVisitor {
```

Comment:
why protected?

Reply:
Remnant of something else I was doing, will change back to private.